js/src/common/utils/mapRoutes.js

```diff
@@ -15,7 +15,7 @@ export default function mapRoutes(routes, basePath = '') {
 
     map[basePath + route.path] = {
       render() {
-        return m(route.component, { routeName: key });
+        return [m(route.component, { routeName: key, key: JSON.stringify(m.route.param()) })];
```

Comment:
Should the routeName be included alongside m.route.param() in the key?

Reply:
Yes. It works without it, but let's include it just in case.